e2e/e2etests/test_precompiles_staking.go

```diff
@@ -0,0 +1,79 @@
+package e2etests
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/stretchr/testify/require"
+
+	"github.com/zeta-chain/node/e2e/runner"
+	"github.com/zeta-chain/node/e2e/utils"
+	"github.com/zeta-chain/node/precompiles/staking"
+)
+
+func TestPrecompilesStaking(r *runner.E2ERunner, args []string) {
```

Comment:
Can we also maybe add a single check somewhere for the share updated using the staking cosmos client instead of the precompile to validate the change at the Cosmos level?